app/javascript/components/workflows/professor/exams/new/editor/components/questions/Matching.tsx

```diff
@@ -1,51 +1,56 @@
-import React, { useState } from 'react';
+import React, { useState, useCallback } from 'react';
```

Comment:
Upon creating a Matching, creating several Column A values and several Column B values, but _not yet setting the correct answers_, autosaving is broken because the exam state doesn't match the schema.  Once one of the answers is non-blank, the schema works again.

Reply:
Unable to repro

Reply:
Is this because you didn't have the schema change?

Reply:
Quite possibly.  Trying again now, and I can't seem to reproduce what I had either.